commons-geometry-core/src/test/java/org/apache/commons/geometry/core/precision/EpsilonDoublePrecisionContextTest.java

```diff
@@ -199,13 +199,13 @@ public void testEquals() {
         EpsilonDoublePrecisionContext c = new EpsilonDoublePrecisionContext(1e-6);
 
         // act/assert
-        Assert.assertFalse(a.equals(null));
-        Assert.assertFalse(a.equals(new Object()));
-        Assert.assertFalse(a.equals(b));
-        Assert.assertFalse(b.equals(a));
+        Assert.assertNotEquals(null, a);
+        Assert.assertNotEquals(new Object(), a);
```

Comment:
JUnit applies its own null checks in the `assertEquals` and `assertNotEquals` methods, meaning that we end up missing those branches in our code. This seems to be why the code coverage decreased. Let's revert these changes on this and all of the other `equals()` method tests and see if the code coverage goes back up.

Reply:
Undid the tests for null checks.